src/main/java/seedu/address/logic/commands/EditSupplierCommand.java

```diff
@@ -117,9 +118,14 @@ private static Supplier createEditedSupplier(Supplier supplierToEdit,
         Address updatedAddress = editSupplierDescriptor.getAddress().orElse(supplierToEdit.getAddress());
 
         supplierToEditOffer = supplierToEdit.getOffers();
```

Comment:
I noticed that you initialised the `supplierToEditOffer` as null, can you remove the initialisation there?

Reply:
Maybe you can move some of the initialisation from execute() to the constructor?